fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -612,7 +612,12 @@ private void addLinkAndOptionsHttpHeaders() {
             servletResponse.addHeader("Accept-Patch", contentTypeSPARQLUpdate);
 
         } else if (resource() instanceof Container) {
-            options = "MOVE,COPY,DELETE,POST,HEAD,GET,PUT,PATCH,OPTIONS";
+            // MOVE/COPY/DELETE operations are not available on a RepositoryRoot resource
+            if (resource().getContainer() == null) {
+                options = "POST,HEAD,GET,PUT,PATCH,OPTIONS";
```

Comment:
so, it _is_ possible to COPY the repository root, but the results are strange. Since it's not strictly prohibited, I'll add the COPY operation back to the allowable methods.